addOns/reports/CHANGELOG.md

```diff
@@ -5,9 +5,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
-### Added
+### Added 
+- The Traditional JSON report, Traditional JSON Report with requests and responses, Traditional XML Report and Traditional XML Report with requests and responses now has "otherinfo" field per alert (Issue 7260).
 - A description of riskdesc fields in the relevant report templates' help (Issue 7445).
 
+### Deprecated
+- The old "otherinfo" entry to be phased out.
```

Comment:
This should be more clear, e.g. `The "otherinfo" under the alert will be removed, superseded by "otherinfo" under the alert instances.`